### ISSUE TITLE
Concerns about the current trusted setup and requests regarding the white paper

### ISSUE THREAD

**Issue by leafcutterant** [AztecProtocol/AZTEC#4]
> Our proof of concept uses a trusted setup generated by our team internally. 

The current on-chain implementation gives no reason to trust it as the trusted setup was generated in a closed fashion. People are very excited about your project, but it should be made clear that they should wait until the public MPC is conducted before they use Aztec or build anything on top of it, should they presume/require anonymity. This would be a good thing to emphasize in the README and the social media posts before someone builds something popular using the current implementation. If you care about the future security of users, please do this.

I also think that, despite Aztec being an anonymity-oriented protocol, the white paper underplays the topic of anonymity and only says "we make this and this assumption". I would like to see explicit discussion of
* privacy attack vectors,
* how statistical methods could reduce the anonymity of the users,
* what an attacker is capable of in Aztec if they successfully reconstruct the toxic waste, and
* what use cases Aztec is *not* suitable for.

Thank you for your work and I'm looking forward to further developments!

**Comment by ArnSch** [comment]
> This would be a good thing to emphasize in the README 
Done, you're right that this needed emphasis. We'll double down on this point once we release the proof generation api.

On the other points, will defer to @zac-williamson

**Comment by zac-williamson** [comment]
Heya, apologies for not getting back to this sooner.

You're correct that we need to describe what you mention in more detail. The paper is a description of the proof system that the AZTEC protocol uses, with formal proofs of soundness etc.

How this proof system is applied to create the AZTEC protocol (and the limitations of this protocol with regards to statistical attacks or improper use of the protocol) needs to be described in more detail, but given the different scope it belongs in a different document. We're putting together an AZTEC token standard that will address these issues and also define the specific interface we use to interact with the proof system described in the paper.

If the toxic waste is recovered, signatures over arbitrary integers can be signed, which would enable an attacker to double spend. I've updated the paper to make this clearer. We've added an emphasis in the readme.md that until we have completed our multiparty computation trusted setup the protocol is very much 'use at your own risk'.

With regards to limitations of the protocol, hopefully [this article](https://medium.com/aztec-protocol/confidential-transactions-have-arrived-a-dive-into-the-aztec-protocol-a1794c00c009) will address some of them until we release our token standard document.
